runtime/src/bank.rs

```diff
@@ -275,6 +275,7 @@ impl Bank {
             bank.update_stake_history(None);
         }
         bank.update_clock();
+        bank.update_rent();
```

Comment:
Probably worth adding/augmenting a bank unit test that covers the fixing of this bug :)

Reply:
done

Reply:
what bug?  rent isn't changing...

Reply:
still curious about what the bug was.  if rent's not changing, no need to update it

Reply:
Issue here is, if we don't do this here, then the bank created from genesis, won't be able to fetch rent account from db.

This statement basically inserts rent sysvar into db at the start.

Reply:
I see.  I thought this was in new_from_parent()